src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+* functionapp: fix app setting issues with Elastic Premium plans
```

Comment:
History and version number need to be cleaned up before we release.

Reply:
Thanks! Please let me know if I need to do anything!

Reply:
You're off the hook, I'll take it from here.